jsonparse.ps1

```diff
@@ -0,0 +1,83 @@
+$location=$args[0]
+$repo=$args[1]
+$name=$args[2]
+
+$location
+$repo
+$name
+$repo.Title
+$JSON = (Get-Content $location) -join "`n" | ConvertFrom-Json
+
+$base = "https://github.com/"
+if($repo.Title -eq "github")
+{
+  $base = "https://github.com/"
+}
+#Renaming the key value of file
+$URI = $JSON.runs
+$JSON.runs.files.PSObject.Properties | foreach-object {
+  $URIReplacement = $_.Name
+  $content = $_.Value
+  $JSON.runs.files.PSObject.Properties.Remove($_.Name)
+  $URIReplacement = $base + $URIReplacement.SubString($URIReplacement.IndexOf($name))
```

Comment:
This doesn't look right. At this point in the code just before this line, the variable `$URIReplacement` has the value `$_`, which is the property name, which is something like `"file:///C:/code/foo.cs"`. And as we've seen, `$name` is really the "project slug", so it has a value like `"Microsoft/sarif-sdk"`.

So unless I'm misreading the code, the string $URIReplacement doesn't contain the string `$name`, so  `$URIReplacement.IndexOf($name)` will return -1.

I might be missing something.